dags.yaml

```diff
@@ -315,3 +315,17 @@ bqetl_desktop_platform:
     retries: 2
     retry_delay: 30m
 
+bqetl_deviations:
+  schedule_interval: 0 4 * * *
+  default_args:
+      owner: ascholtz@mozilla.com
+      start_date: '2020-10-01'
```

Comment:
The DAG got removed on 2020-10-02. The datasets are still there. Once this gets merged I'll run a backfill to update the dataset.